src/components/Button/Button.tsx

```diff
@@ -0,0 +1,30 @@
+
+import React from "react";
+import './Button.css'
+
+interface ButtonProps {
+  submit: () => void;
+  buttonText?: string; 
+}
+
+
+
+export const Button = ({submit, buttonText}: ButtonProps) =>{
+return (
+  // modal__form-submit
+  <div className="modal__form-submit">
+          <button
```

Comment:
To add on to Ruven's hover style comment, I did notice that when hovering on the button the pointer seemed a little off. It only appeared on the bottom portion of the button.